app/controllers/content_types_controller.rb

```diff
@@ -11,15 +11,15 @@ def show
 
   # set the content type in the content metadata
   def update
-    authorize! :manage_item, @cocina_object
+    authorize! :manage_item, @cocina
```

Comment:
what was setting up `@cocina_object` before?  i can see how `@cocina` gets set up, just wondering how the old instance variable was working (was curious since i didn't notice its setup being removed in this PR).

Reply:
oh nm, bad read on my part